doc/classes/KinematicBody2D.xml

```diff
@@ -80,6 +86,13 @@
 			<argument index="4" name="floor_max_angle" type="float" default="0.785398">
 			</argument>
 			<description>
+				Moves the body along the given vector. [code]linear_velocity[/code] should be given in pixels per second. If the body collides with another body, it will slide along the other body rather than stop immediately. Additionally, if the floor the body stands on is also a KinematicBody2D, the body will move together with the floor. This works regardless of the method used to move the floor, and is useful for making moving or rotating platforms in a platformer game.
```

Comment:
Suggestion:
`Moves the body along a vector. [code]linear_velocity[/code] is a value in pixels per second. If the body collides with another, it will slide along the other body rather than stop immediately. If the floor the node stands on is also a [code]KinematicBody2D[/code], the node will follow the floor's motion. This works regardless of the method used to move the floor. You can use it to make moving or rotating platforms.`